src/docs/reference/support-policy.md

```diff
@@ -126,8 +126,8 @@ please see [this document](https://github.com/ionic-team/stencil/blob/main/docs/
 
 #### Puppeteer
 
-| Stencil Version | Puppeteer v5-9 | Puppeteer v10 | Puppeteer v11 | Puppeteer v12 | Puppeteer v13 |
-|:---------------:|:--------------:|:-------------:|:-------------:|:-------------:|:-------------:|
-|       V3        |   &#10060;     |   &#9989;     |   &#9989;     |   &#9989;     |   &#9989;     |
-|       V2        |   &#9989;      |   &#9989;     |   &#10060;    |   &#10060;    |   &#10060;    |
-|       V1        |   &#9989;      |   &#10060;    |   &#10060;    |   &#10060;    |   &#10060;    |
+| Stencil Version | Puppeteer v5-9 | Puppeteer v10 | Puppeteer v11-18 | Puppeteer 19 |
```

Comment:
we've got a little inconsistency here, if we want to stick with what we've been doing maybe this?

```suggestion
| Stencil Version | Puppeteer v5-9 | Puppeteer v10 | Puppeteer v11-18 | Puppeteer v19 |
```

or we could change them all to `Puppeteer 5-9`, `Puppeteer 10`, etc

Reply:
@alicewriteswrongs I added 'v' in https://github.com/ionic-team/stencil-site/pull/932/commits/d5f3cea41c422b82c05db24c3930989cf5cb11ac and fixed up the table alignment as a result in that same commit